app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Api
+  module V1
+    class UsersController < Api::V1::ApiController
+      def show
+        @user = User.find(params[:id])
+        authorize @user
+        render 'users/show', locals: { user: @user }
```

Comment:
drop this line and the controller will try to render the corresponding view in the folder